app/controllers/posts_controller.rb

```diff
@@ -14,7 +14,7 @@ def new
   def edit; end
 
   def create
-    @post = Posts::Create.new(post_params).call
+    @post = Posts::Create.new(post_params.merge(user: current_user)).call
```

Comment:
można również ten merge robić w samej metodzie post_params, wydaje mi się że byłoby to troszku lepsze